docker-compose.yml

```diff
@@ -111,38 +112,61 @@ services:
       - postgres:database.service.rasterfoundry.internal
 
   backsplash:
-    image: openjdk:8-jre
```

Comment:
This image tag is negated by the `build` block below. 

What this actually does, is build an image using `Dockerfile.backsplash` and tag it as `openjdk:8-jre`.